etc/ncp-config.d/nc-prettyURL.sh

```diff
@@ -16,6 +16,13 @@ OCC="$NCDIR/occ"
 
 install() { :; }
 
+isactive()
+{
+  local REWRITEBASE
+  REWRITEBASE="$( grep RewriteBase /var/www/nextcloud/.htaccess )" || return 1
```

Comment:
cool, looks good

so when it is disabled, `Rewritebase` disappears from `.htaccess`, right?

have you tested it?

Reply:
I couldn't test this yet. We might need to check the value, but I yet have to test it.

Reply:
`RewriteBase` is indeed deleted from `.htaccess`. Hence, the function is correct.